malcolm/core/statemachine.py

```
@@ -29,7 +37,7 @@ def __init__(self, name):
         self.possible_states.append(self.DISABLED)
 
     def create_states(self):
-        raise NotImplementedError()
+        pass
```

Is this change for a specific reason? Seems like it could cause silent errors.